hunting/index.md

```diff
@@ -0,0 +1,8 @@
+# List of Available Queries
+
+Here are some of the queries currently available:
+
+## llm
+- [Denial of Service or Resource Exhaustion Attacks Detection](llm/rules/docs/llm_dos_resource_exhaustion_detection.md)
```

Comment:
oooh - this makes me wonder if we should include the language in the metadata too .... the `os_query` could just be another language. Then I would update the script to add it here 

```suggestion
- [Denial of Service or Resource Exhaustion Attacks Detection](llm/rules/docs/llm_dos_resource_exhaustion_detection.md) (ES|QL)
```

Reply:
Loved this idea. Added.